metals/src/main/scala/scala/meta/internal/parsing/FoldingRangeExtractor.scala

```diff
@@ -183,7 +165,16 @@ final class FoldingRangeExtractor(
 
         case c: Case =>
           val startingPoint = c.cond.getOrElse(c.pat)
-          val bodyEnd = c.body.pos.end + 1
+          val bodyEnd = {
+            c.body.pos match {
+              case Position.None => c.body.pos.end
+              case Position.Range(input, _, end) =>
+                val nextChar = input.chars(end + 1)
```

Comment:
Oh. I missed test symbol idx changes.
It's needed to tests `end -1` because `Position.end` isn't inclusive